nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees_desc_asb_[7,).partial.md

```diff
@@ -0,0 +1,15 @@
+In NServiceBus version 6 and above, the shape of the pipeline has changed so that a transport is in full charge of how a given transaction mode is implemented. A transaction scope is no longer mandatory in order to orchestrate dispatching of messages with complete/rollback behavior. The Azure Service Bus transport only uses a transaction scope to implement the `SendAtomicWithReceive` transaction mode. Other modes work without any scope at all. The new architecture for `SendAtomicWithReceive` is schematically represented like this:
+
+![Transactions v7](transactions-v7.png)
```

Comment:
Indeed, the pipline consists of sections, with forks in between, sends do not happen as part of the user code's pipeline section, but wait until the complete execution of that part of the pipe has returned